org.lflang/src/lib/cpp/time_parser.hh

```diff
@@ -131,6 +151,8 @@ std::string validate_time_string(const std::string& time) {
     return "The empty string is not a valid time!";
   } else if (trimmed[0] == '-') {
     return "Negative values are not a valid time!";
+  } else if (iequals("forever", time)) {
+    return ""; // "forever" is a valid value
```

Comment:
Is the value "forever" defined by LF or by the C++ target only? It's the first time I see this.

Reply:
Forever is also used in the C target. I am not sure if it is specified anywhere to be honest.